src/test/scala/io/iohk/ethereum/vm/MockWorldState.scala

```diff
@@ -62,4 +62,8 @@ case class MockWorldState(
   def isAccountDead(address: Address): Boolean =
     getAccount(address).forall(_ == Account.empty())
 
+  def resetAccount(address: Address): MockWorldState = {
+    val account = getGuaranteedAccount(address).resetAccount()
+    saveAccount(address, account)
+  }
```

Comment:
Minor: since it's written in terms of other methods defined in this class, it could be moved to `WorldStateProxy`, and real and mock implementations would not need to override.